clang/test/Driver/sycl.c

```diff
@@ -72,7 +72,7 @@
 // HEADER_ORDER-NOT: clang{{.*}} "/usr/include"{{.*}} "-internal-isystem" "{{.*}}bin{{[/\\]+}}..{{[/\\]+}}include{{[/\\]+}}
 
 /// Verify -fsycl-device-only phases
-// RUN: %clang -### -ccc-print-phases -fsycl-device-only %s 2>&1 | FileCheck %s --check-prefix=DEFAULT-PHASES
+// RUN: %clang -### -ccc-print-phases -target x86_64-unknown-linux-gnu -fsycl-device-only %s 2>&1 | FileCheck %s --check-prefix=DEFAULT-PHASES
```

Comment:
Just wondering, how has the absence of the `--target` argument been affecting the `DEFAULT-PHASES` checks? Given the absence of host triple mentions in these.

Reply:
The running on from a 32-bit build, the default target is 32-bits, causing the default `sycldevice` to be `spir` instead of `spir64`, which causes the failure.